examples/jsm/renderers/webgpu/WebGPURenderer.js

```diff
@@ -23,7 +23,7 @@ class WebGPURenderer {
 		// public
 
 		this.domElement = ( parameters.canvas !== undefined ) ? parameters.canvas : document.createElementNS( 'http://www.w3.org/1999/xhtml', 'canvas' );
-		this.parameters = parameters;
+		this.parameters = Object.assign( {}, parameters );
```

Comment:
Do we need this code although the class already uses a default parameter? Meaning:
```js
constructor( parameters = {} ) {
```

Reply:
[I update some fields of `parameters` to ensure](https://github.com/mrdoob/three.js/pull/20281/files#diff-b2e471696b2cc2a9842ff64fe2ec36bbR166). I want to avoid having an effect to object user passed. So it is for copying object. But no strong opinion.

Reply:
Ah, I see now the reason for this! Make sense 👍 .